### PR TITLE
Feature/invalid json fix

### PR DESCRIPTION
**Pull request by abhishekjain88**

### CODE REVIEW

**Comment by bargenilesh** [comment]
@abhishekjain88 Looks great 👍  Thanks for adding "NoOpEventsIngestionExecutor" 👍 
:shipit: Provided tests are passed...

File: modules/event/src/test/java/com/intuit/wasabi/events/impl/NoOpEventsIngestionExecutorTest.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import com.intuit.wasabi.eventobjects.EventEnvelopePayload;
+import com.intuit.wasabi.events.EventIngestionExecutor;
+import org.junit.Test;
+
+import static io.codearte.catchexception.shade.mockito.Mockito.mock;
+import static io.codearte.catchexception.shade.mockito.Mockito.when;
+import static junit.framework.TestCase.assertEquals;
+
+public class NoOpEventsIngestionExecutorTest {
```
Comment (iizrailevsky) [inline modules/event/src/test/java/com/intuit/wasabi/events/impl/NoOpEventsIngestionExecutorTest.java]: @abhishekjain88 Thanks for adding the test! 👍

File: modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java [hunk 2]
```diff
@@ -105,7 +105,7 @@ public String toJson() {
                 event != null && event.getTimestamp() != null ? event.getTimestamp().getTime(): null);
         eventJson.put("eventType", event != null ? event.getType() + "": "null");
         eventJson.put("eventName", event != null ? event.getName() + "": "null");
-        eventJson.put("eventPayload", event != null ? event.getPayload(): null);
+        eventJson.put("eventPayload", event != null && event.getPayload() != null ? event.getPayload().toString(): null);
```
Comment (iizrailevsky) [inline modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java]: @abhishekjain88 Mind adding a quick comment to indicate why we need to do ".toString()"?

Comment (abhishekjain88) [inline modules/event-objects/src/main/java/com/intuit/wasabi/eventobjects/EventEnvelopePayload.java]: @iizrailevsky Sounds good! done!

**Pull request merged**